File: Repeet/Models/Flashcard.cs
namespace Repeet.Models
{
    public class Flashcard
    {
        public Guid Id { get; set; }
        public string Keyword { get; set; } = string.Empty;
        public string Definition { get; set; } = string.Empty;
        public Guid? SetId { get; set; }
        public Set? Set { get; set; }
        public Guid? OwnerId { get; set; }
        public User? Owner { get; set; }
    }
}

File: frontend/src/Models/Flashcard.ts
export type FlashcardGet = {
    id: string;
    keyword: string;
    definition: string
};

export type FlashcardPost = {
    keyword: string,
    definition: string,
    setId: string
}

export type FlashcardPut = {
    keyword: string,
    definition: string
}